metric/blockgroup/mUNFRENT-accuracy.csv

```diff
@@ -1,154 +1,154 @@
-370630020222,48.73701983,36.15995224,34.47144472,31.7302257,25.42396527,19.56827272,22.1252127599607,25.43088817879
-370630020231,41.37073189,32.75527005,28.6044489,29.23046009,28.13645322,16.32487542,16.7195262391297,18.2942590734537
-370630020232,35.32268378,22.42760107,19.93329212,13.96695997,14.61005433,15.12079834,16.8325822853327,14.3964099094992
-370630020241,53.01273773,29.21891899,14.73326614,28.31852809,24.80559897,23.10184557,22.0298576357503,27.337874324739
-370630020242,64.84865905,42.90591814,41.01665704,37.28510627,31.2356188,13.73818582,14.3762906540542,17.8321141729386
-370630020243,52.94526669,21.14089297,20.28885229,22.32024335,27.12203006,41.31687213,43.1553977487167,42.019154858231
-370630020251,187.5609065,81.25,17.32050808,48.73713822,18.31115219,18.30604408,42.9283473156367,41.2108811628862
-370630020252,47.52798672,34.42733618,32.93219057,42.54118797,34.89311176,34.16268404,29.5792937010649,22.3397981872324
-370630020253,53.24773806,29.02021052,31.6348342,10.74758454,16.16875267,17.31479896,23.9495900707845,7.14285714285714
-370630020261,26.80746641,7.933537363,23.26231208,21.11627251,24.13877327,24.17358683,27.7090907642287,22.8561693513825
-370630020262,18.89105367,18.99465773,17.36252635,17.55558229,15.87339369,14.95481663,19.5590074946388,15.969939415936
-370630020271,49.7483458,15.05755128,15.82326378,12.89626172,12.75931884,50.94866833,42.7298300248723,42.7682338657037
-370630020272,18.21323137,17.72586385,19.13258702,18.05565392,17.49505045,16.28492324,17.3436538231089,16.4628540351895
-370630020273,15.73812122,17.36347015,14.76871881,13.9698583,16.86472796,16.21607433,20.4162349246951,18.2148739781797
-370630020281,147.516059,45.38630098,32.50202211,33.92804119,27.52435798,23.00079528,24.7067347200066,22.7238937129654
-370630020282,20.0790991,13.10417263,10.95429146,8.617242576,10.4368623,15.48141637,15.8748053164218,15.6146289230989
-370630020283,11.18186055,9.337881552,10.50710532,11.97221396,14.49932007,11.46461158,12.0897362403739,11.9009744030715
-370630021001,188.5081406,38.32222446,45.44556941,27.16942443,49.26254239,45.89392237,67.3796676094251,38.3786718850158
-370630021002,83.46616694,40.84666743,53.56735474,78.67714853,76.72674832,52.10144984,48.0758475615562,40.5978949971043
-370630021003,106.6548708,31.85961974,35.52942223,65.87909797,74.80962807,72.18162273,77.1357116838583,40.6058778769512
-370630022001,22.27882011,12.99322133,10.84015761,11.09328707,9.296149917,9.11298339,10.3652860775217,11.8732357101498
-370630023001,18.07703497,20.73347089,22.81152461,21.10592019,17.53896792,20.62808672,32.5942919376448,22.2804421934026
-370630023002,319.2184217,40.53893441,20.39576453,32.08990142,31.05912285,24.10559948,26.5271550210798,15.1925633349006
-370639801001,,,,,,,,
+"id","y_2011","y_2012","y_2013","y_2014","y_2015","y_2016","y_2017","y_2018","y_2019"
+370630001011,56.20680272,26.57041309,24.28510594,28.47672003,26.10530482,26.93677964,27.2842631665674,27.2546690037882,53.1839614057274
```

Comment:
@BiSeye The new values here are consistently much higher than prior years. This makes me wonder if the numerator (estimate) used for the calculation might have been one of the renter categories but not all? For example renters paying 30-34% of their income on housing costs (B25070_007), but not the other categories (B25070_008, B25070_009, etc).